### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by terichadbourne**
partially addresses #69

### CODE REVIEW

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!

File: README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 🚧 Under construction
```
Comment (jessicaschilling) [inline README.md:7]:
```suggestion
```
Might as well remove the construction banner too.

**Review by jessicaschilling — CHANGES_REQUESTED** [review]
We should remove the construction banner, but otherwise good to merge.

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!

**Pull request merged**